fhir-server/liberty-config/config/default/fhir-server-config-postgresql.json

```diff
@@ -30,10 +30,10 @@
                 "default": {
                     "type": "postgresql",
                     "connectionProperties": {
-                        "serverName": "localhost",
+                        "serverName": "postgres_postgres_1",
                         "portNumber": 5432,
-                        "databaseName": "fhirdata",
-                        "user": "fhirserver",
+                        "databaseName": "fhirdb",
+                        "user": "fhiradmin",
```

Comment:
as per albert's comment, this doesn't seem to match the user created  `build/persistence/postgres/resources/db.sql` ...how does it work?

Reply:
now I see that it:
1. creates this system user in the dockerfile; and
2. creates this fhiradmin db user as the FHIRDB owner in the entry-point https://github.com/IBM/FHIR/pull/1234/files#diff-6d0acfaa35776bfcbc9f9cec15f60f6dR40

do the users created in db.sql still work?

Reply:
yeah - they still work.  I'll clear this up later on, but great points, I only meant FHIRADMIN to deploy the schema, and fhirserver was meant to test/run it

Reply:
I'm trying this out.  I had meant to keep fhirserver.  I'm going to check/test this, and update the Dockerfile to reflect it.

Reply:
Cleaned this up so we create the users as much as possible in the dockerfile.